internal/graph/check.go

```diff
@@ -411,6 +415,10 @@ func exclusion(ctx context.Context, concurrencyLimit uint32, handlers ...CheckHa
 		}
 	}
 
+	if errs != nil {
```

Comment:
If you're using `multierror` there's a chance that you don't actually have any errors but `errs` is not nil, so you'll enter this condition erroneously. This is due to the fact that `errs.Errors` could be instantiated as an empty slice. 

To prevent this, use `errs.ErrorOrNil() != nil `. 

However we could also remove this dependency entirely and use https://pkg.go.dev/errors#Join instead in the future.

Reply:
That would be good 👍

Reply:
Fixed up. Thanks!